bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -450,27 +587,31 @@ impl From<ffi::evmc_result> for ExecutionResult {
     }
 }
 
-fn allocate_output_data(output: Option<&Vec<u8>>) -> (*const u8, usize) {
+fn allocate_output_data<T>(output: Option<&Vec<T>>) -> (*const T, usize) {
```

Comment:
I think this makes sense, but I am a little concerned about merge issues when upgrading evmc. Shall we make a second function signature for the new allocations that you need?

Reply:
Sure we could do that. @HerbertJordan what do you think?

Reply:
No, overloading is not possible. The only possibility would be to have two functions. One for `u8` and one for `Uint256`. Or we stick with the generic one.